src/Connection.php

```diff
@@ -299,6 +299,12 @@ class Connection extends Component
      * @since 2.0.12
      */
     public $useSSL = false;
+    /**
+     * @var array PHP context options https://www.php.net/manual/en/context.ssl.php which are used in the Redis connection stream.
+     * @since 2.x
+     */
+
+    public $contextOptions = [];
```

Comment:
```suggestion
    /**
     * @var array PHP context options which are used in the Redis connection stream.
     * @see https://www.php.net/manual/en/context.ssl.php
     * @since 2.0.15
     */
    public $contextOptions = [];
```